.buildkite/scripts/steps/functional/security_serverless_playwright.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/scripts/steps/functional/common.sh
+
+export JOB=kibana-security-solution-playwright
+export KIBANA_INSTALL_DIR=${KIBANA_BUILD_LOCATION}
+
+echo "---SERVERLESS - Security Solution Playwright Tests"
+
+yarn playwright install
```

Comment:
We should add this to `yarn kbn bootstrap` so it can be cached on CI, or use the already installed system browsers.

Reply:
@jbudz can you please point me out where/how to do so? I tried to do it myself in this [commit](https://github.com/elastic/kibana/pull/190803/commits/aa81ff50ae330df5bef9b2f864efebc753d300a5) but the test is failing, so I'm assuming I did it in the wrong way.

Reply:
*UPDATE:* I found the issue that was making the test fail, was not related with those changes.

@jbudz would you mind giving another look at the PR? If I'm not mistaken I already addressed all your comments. Thanks! :)